rerun_cpp/src/rerun/error.hpp

```diff
@@ -30,6 +30,7 @@ namespace rerun {
         Ok = 0x0000'0000,
         OutOfMemory = 0x0000'0001,
         NotImplemented = 0x0000'0002,
+        InvalidOperationOnMovedObject = 0x0000'0003,
```

Comment:
what is this used for? maybe add a docstring explaining it?

Reply:
no longer need it now. @Wumpf plz remove